peekaboo/toolbox/file.py

```diff
@@ -58,18 +58,18 @@ def guess_mime_type_from_filename(self, file_path):
     def guess_mime_type_text_representation(self, file_path):
         """ Guess the type by content and hand back text representation rather than
         mime type. """
-        typeAsText = magic.from_file(file_path)
```

Comment:
Seeing this camel-case change I guess this was mainly to make pylint happy, right?

Reply:
Addressed in #143 by squashing and already inserting the variable name non-camel-cased.